torch/utils/tensorboard/proto_graph.py

```diff
@@ -0,0 +1,39 @@
+from tensorboard.compat.proto.node_def_pb2 import NodeDef
+from tensorboard.compat.proto.attr_value_pb2 import AttrValue
+from tensorboard.compat.proto.tensor_shape_pb2 import TensorShapeProto
```

Comment:
Is this module public? If it is, what does it do? If it is not, please prefix with an underscore.

Reply:
@lanpa please respond with some explaination on what this does, but I've moved it to `_proto_graph.py`.

Reply:
"""
This module contains the functions needed to convert pytorch graph
into tensorboard GraphDef proto.
It uses torch's jit trace to get the graph and followed by parsing
graph components into tensors and operations. Nodes that lack of
scope information will be assigned one in a hueristic way. Finally,
the parsed graph will be wrapped into GraphDef and return to caller.
"""